### ISSUE TITLE
Version-control current set of Ground Systems' Grafana dashboards.

### ISSUE THREAD

**Issue by harrisonCassar** [UCLA-Rocket-Project/AresSoftware#24]
# Motivation/Description
Currently, all created Grafana Dashboards live on the RPi, and are only accessible/viewable if the Grafana service is running on the RPi, and an external user connects to Grafana via the connected network. Any changes that are made to these Dashboards are not documented in any way (other than Grafana's built-in temporary revision history support). We want to be able to version-control any changes that we make to these dashboards to not only have easier tracking of these changes, but also enable code review to be completed.

# Work To-Be-Done
Extract Grafana Dashboards from RPi's Grafana instance, and add/upload them to this repository in a subdirectory that makes sense (perhaps a child of the `Ground Systems` directory?).

Blocked by #9.

**Comment by philipdoucla** [comment]
Moved to Low Priority due to other more important work in the pipeline (config files (TO BE TICKETED), and deployment (#16)).

No current significant risk to system operation due to Grafana's role as a data viewer (never modifies data).

~TRC 1/23/23
